Makefile

```diff
@@ -76,14 +79,6 @@ verify_installed:
 	@@${PERL} -V:'install.*'
 	# ensure cpan is defined
 	@@${PERL} -MCPAN -e 'print $$CPAN::VERSION . "\n";'
-	# ensure sqitch is >= 0.97.0
```

Comment:
We can only check that after install then. Because if there's another `sqitch` installed we have no (easy) way to tell if the one installed by `cpanminus` with have the priority beforehand, right?

Reply:
yessir